src/test/java/dev/braintrust/TestHarness.java

```diff
@@ -111,6 +113,30 @@ public List<SpanData> awaitExportedSpans() {
         return spanExporter.getFinishedSpanItems();
     }
 
+    /**
+     * flush all pending spans and return all spans which have been exported so far
+     *
+     * <p>repeat the process until the number of exported spans equals or exceeds `minSpanCount`
+     */
+    @SneakyThrows
+    public List<SpanData> awaitExportedSpans(int minSpanCount) {
+        var spans = awaitExportedSpans();
+        int attempts = 0;
+        while (spans.size() < minSpanCount) {
+            attempts++;
+            if (attempts > 30) {
+                fail(
+                        String.format(
+                                "Timeout waiting for spans: expected at least %d spans, but got %d"
+                                        + " after %d attempts",
+                                minSpanCount, spans.size(), attempts));
+            }
+            Thread.sleep(1000);
```

Comment:
Why do you need to wait & sleep? To read off the OTel thread? Is there a faster, more directly way to do this synchronously in the test suite?